diff -urN wmmail-0.59.orig/global.c wmmail-0.59/global.c
--- wmmail-0.59.orig/global.c	Wed May 13 14:23:11 1998
+++ wmmail-0.59/global.c	Mon Dec  7 10:04:40 1998
@@ -49,6 +49,7 @@
 
 char *app_name            = NULL;
 char *config_file_path    = NULL;
+char *global_config_file_path	= "/usr/share/wmmail/wmmailrc";
 char *exec_on_click_path  = NULL;
 char *exec_on_new_path    = NULL;
 char *exec_on_update_path = NULL;
diff -urN wmmail-0.59.orig/sample.wmmailrc wmmail-0.59/sample.wmmailrc
--- wmmail-0.59.orig/sample.wmmailrc	Wed May 13 14:23:11 1998
+++ wmmail-0.59/sample.wmmailrc	Mon Dec  7 10:00:28 1998
@@ -23,7 +23,7 @@
 #########################################################
 # Program to execute when left mouse button is pressed	#
 #########################################################
-Execute "xterm -T Pine -e pine -i &"
+Execute "hanterm -T Mutt -e mutt &"
 
 
 #########################################################
@@ -101,18 +101,18 @@
 # Assign icons/animations				#
 #########################################################
 NoMail  
-        /usr/include/X11/pixmaps/Mail1.xpm
+        /usr/share/wmmail/pixmaps/Mail1.xpm
 End
 
 OldMail         
-        /usr/include/X11/pixmaps/Mail1.xpm
+        /usr/share/wmmail/pixmaps/Mail1.xpm
 End
 
 NewMail  
-        /usr/include/X11/pixmaps/Mail1.xpm
-        /usr/include/X11/pixmaps/Mail2.xpm
-        /usr/include/X11/pixmaps/Mail3.xpm
-        /usr/include/X11/pixmaps/Mail2.xpm
+        /usr/share/wmmail/pixmaps/Mail1.xpm
+        /usr/share/wmmail/pixmaps/Mail2.xpm
+        /usr/share/wmmail/pixmaps/Mail3.xpm
+        /usr/share/wmmail/pixmaps/Mail2.xpm
 End
 
 #########################################################
diff -urN wmmail-0.59.orig/wmmail.c wmmail-0.59/wmmail.c
--- wmmail-0.59.orig/wmmail.c	Wed May 13 14:23:11 1998
+++ wmmail-0.59/wmmail.c	Mon Dec  7 10:05:06 1998
@@ -137,8 +137,11 @@
   /* open configuration file */
   if ((yyin = fopen(config_file_path, "r")) == (FILE *) NULL)
   {
-    croak("cannot open file %s", config_file_path);
-    exit(ERR_IO);
+    if ((yyin = fopen(global_config_file_path, "r")) == (FILE *) NULL)
+    { 
+      croak("cannot open file %s", config_file_path);
+      exit(ERR_IO);
+    }
   }
 
   /* parse configuration file, then either proceed or quit */
diff -urN wmmail-0.59.orig/wmmail.h wmmail-0.59/wmmail.h
--- wmmail-0.59.orig/wmmail.h	Wed May 13 14:23:11 1998
+++ wmmail-0.59/wmmail.h	Mon Dec  7 09:52:18 1998
@@ -141,6 +141,7 @@
 
 extern char *app_name;
 extern char *config_file_path;
+extern char *global_config_file_path;
 extern char *exec_on_click_path;
 extern char *exec_on_update_path;
 extern char *exec_on_new_path;
